concourse/scripts/val_from_yaml.rb

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
```

Comment:
/cc @Jonty 

> I assume this script has been grabbed from somewhere online, would be nice to have the link in the commit message for future reference.

Err, yes and no. I started with https://github.com/alphagov/cf-terraform/blob/master/scripts/val_from_yaml.rb (which I wrote back then), but I re-factored it so much, and I added a spec that I don't know if it is even worth it to refer to it.